src/server/page.ts

```diff
@@ -212,8 +212,11 @@ export class Page extends SdkObject {
   }
 
   _didClose() {
+    if (this._closedState === 'closed') {
```

Comment:
I'd rather not change this, but remove the background page from `CRBrowser._backgroundPages` collection.

Reply:
We unfortunately have to emit the close event so it reaches the client.